docs/myst/notebook-text.md

```diff
@@ -0,0 +1,61 @@
+---
+# A Jupyter Notebook written in Markdown.
+# https://myst-nb.readthedocs.io/
+file_format: mystnb
+---
+
+(notebook-text)=
+
+# Notebook (text-based)
+
+The documentation can include [text-based notebooks] using [MyST-NB],
+effectively bringing Jupyter technologies to Markdown.
+
+## Basics
+
+```{code-cell} ipython3
+import sys
+print("this is some stdout")
+print("this is some stderr", file=sys.stderr)
+```
+
+:::{tip}
+See also {ref}`notebook-traditional` and {ref}`cells`.
+:::
+
+## SQL Magics
+
+[JupySQL], the successor of [ipython-sql], enables running SQL in Jupyter/IPython
+via `%sql` and `%%sql` magics.
+
+```{code-cell} ipython3
+# Acquire data.
+!pip --quiet install csvkit
+!curl -s -L -O https://github.com/wireservice/csvkit/raw/refs/heads/master/examples/realdata/acs2012_5yr_population.csv
+!rm -f population.db
+!csvsql --db sqlite:///population.db --insert acs2012_5yr_population.csv
+```
+```{code-cell} ipython3
+# Run query using JupySQL.
+%reload_ext sql
+%sql sqlite:///population.db
+%sql SELECT * FROM acs2012_5yr_population ORDER BY total_population DESC LIMIT 10;
+```
```

Comment:
Thanks for spotting the dark mode bug here, @surister.

![image](https://github.com/user-attachments/assets/9b149fc3-8800-4e91-95fc-bd2a55673b49)

Reply:
Thanks to @msbt's excellent suggestion, submitting the fix 6221203ef8ae201 was easy, and it solves the problem perfectly well. Thank you very much. 💯 

![image](https://github.com/user-attachments/assets/9905d830-7f51-4589-a023-2a2bdb86df4a)

Reply:
To conclude this, filing an issue with [MyST-NB](https://github.com/executablebooks/myst-nb) about it would be perfectly sensible, sharing our little CSS snippet.

Reply:
> [...] filing an issue with MyST-NB about it would be perfectly sensible [...]

In the spirit of upstreaming changes, let me share a few pointers to existing issues about dark mode already (I haven't digested them, so it's really just for reference and information sharing). Also having backlinks on them can help others to resolve the problem in an ad hoc manner, and evaluate our solution, until a new release of MyST-NB will include a relevant fix.

- https://github.com/executablebooks/MyST-NB/issues/523
- https://github.com/executablebooks/MyST-NB/issues/607
- https://github.com/executablebooks/MyST-NB/issues/618